### ISSUE TITLE
Softmax, MLP, CNN 三种方法识别手写数字MNIST——《TensorFlow 实战》读书笔记 | S1NH

### ISSUE THREAD

**Issue by duchengyao** [duchengyao/comment#5]
http://localhost:4000/post/Tensorflow-MNIST/

**Comment by duchengyao** [comment]
hy
